app/assets/javascripts/analyses.coffee

```diff
@@ -0,0 +1,15 @@
+$ ->
+  new GwasAnalysis('.new-gwas-analysis').init()
+  new ManhattanPlot('#gwas-manhattan-plot').init()
+  new AnalysisRefresher('.show-analysis').init()
+  new AnalysisListRefresher('.analyses').init()
+
+  data_table = $('#analysis_results table').DataTable(
+    columnDefs: [
+      { type: "num", targets: 3 }
+    ]
+  )
+
+  # NOTE: No idea why, but it does not work if included in data table
+  # definition above. Perhaps that's how it works with prepopulated tables?
+  data_table.order([3, 'desc']).draw()
```

Comment:
I think it does - I pushed a change. your culprit is probably your local storage. Go to the 'Local storage' of your browser and remove anything related to BIP analyses. DataTables store your last column of order and uses that, instead of the configuration, the next time you see the table.